README.md

```diff
@@ -1,4 +1,5 @@
-![AstroVascPy Logo](docs/source/logo/BBP-AstroVascPy-Github.jpg)
+<img src="https://github.com/BlueBrain/AstroVascPy/blob/main/docs/source/logo/BBP-AstroVascPy-Github.jpg?raw=true" alt="AstroVascPy Logo"/>
```

Comment:
Why this change? This will unconditionally refer to the image in the default branch and makes gauging changes harder.

Reply:
https://github.com/BlueBrain/AstroVascPy/blob/c2ac01ad5d9d794d84ea4001dccea84948589689/docs/source/index.rst?plain=1#L1

Here we include the `readme.md`. 
But the generated website [astrovascpy.rtfd.io](https://astrovascpy.rtfd.io/) does not show the logo.  
That change fixes the problem, but I don't know if there is a better way. Any suggestion?   (also, at the moment I don't have access to the building process)

Reply:
Ok, maybe I have another idea.